src/octree/raytracing/classic_raytracing_on_bevy_wgpu.rs

```diff
@@ -46,10 +46,10 @@ impl<T: Default + Clone + VoxelData, const DIM: usize> Octree<T, DIM> {
                                 let content = data[x][y][z].user_data();
                                 voxels.push(Voxelement {
                                     albedo: Color::rgba(
```

Comment:
@davids91 For this we could create a simple From<shocovox::Albedo> for bevy::Color

Reply:
Yes, that's def a great addition; Fits into the bevy support topic as well